library/kernel_settings.py

```diff
@@ -171,42 +171,53 @@ def wrapper_set_logger_class(_clsname):
 EXAMPLES = """
 # add the sysctl vm.max_mmap_regions, and disable spectre/meltdown security
 kernel_settings:
-    sysctl:
-      - name: vm.max_mmap_regions
-        value: 262144
-    sysfs:
-      - name: /sys/kernel/debug/x86/pti_enabled
-        value: 0
-      - name: /sys/kernel/debug/x86/retp_enabled
-        value: 0
-      - name: /sys/kernel/debug/x86/ibrs_enabled
-        value: 0
+  sysctl:
+    - name: vm.max_mmap_regions
+      value: 262144
+  sysfs:
+    - name: /sys/kernel/debug/x86/pti_enabled
+      value: 0
+    - name: /sys/kernel/debug/x86/retp_enabled
+      value: 0
+    - name: /sys/kernel/debug/x86/ibrs_enabled
+      value: 0
 
 # replace the existing sysctl section with the specified section
 # delete the /sys/kernel/debug/x86/retp_enabled setting
 # completely remove the vm section
 # add the bootloader cmdline arguments spectre_v2=off nopti
 # remove the bootloader cmdline arguments quiet splash
 kernel_settings:
-    sysctl:
-      - previous: replaced
-      - name: vm.max_mmap_regions
-        value: 262144
-    sysfs:
-      - name: /sys/kernel/debug/x86/retp_enabled
-        state: absent
-    vm:
-      state: empty
-    bootloader:
-      - name: cmdline
-        value:
-          - name: spectre_v2
-            value: off
-          - name: nopti
-          - name: quiet
-            state: absent
-          - name: splash
-            state: absent
+  sysctl:
+    - previous: replaced
+    - name: vm.max_mmap_regions
+      value: 262144
+  sysfs:
+    - name: /sys/kernel/debug/x86/retp_enabled
+      state: absent
+  vm:
+    state: empty
+  bootloader:
+    - name: cmdline
+      value:
+        - name: spectre_v2
+          value: off
+        - name: nopti
+        - name: quiet
+          state: absent
+        - name: splash
+          state: absent
+"""
+
+RETURN = """
+message:
```

Comment:
I believe that by convention this is called `msg`. https://docs.ansible.com/ansible/latest/reference_appendices/common_return_values.html#msg . Unless you want to keep `msg` only for error messages, but I do not see any prior art for this in existing core modules.

Reply:
fixed in https://github.com/linux-system-roles/kernel_settings/pull/3/commits/3d6c2d7828fa164c6fe89f5f6fdc3066218dd289

Reply:
sorry, fixed in https://github.com/linux-system-roles/kernel_settings/commit/3d6c2d7828fa164c6fe89f5f6fdc3066218dd289 and https://github.com/linux-system-roles/kernel_settings/pull/3/commits/20646a90b2043059fe7a890da8022459384b2160